common/script/libs/achievements.js

```diff
@@ -0,0 +1,507 @@
+import i18n from '../i18n';
+
+let achievs = {};
+let index;
+
```

Comment:
I don't really understand what do you mean by content files, since there are no achievements in the content api result.

At first I had this logic inside a class, which would be created on every request.

Or do you mean the functions in this file? Anyway ping me on Slack ^^